core/src/main/java/org/frankframework/parameters/ParameterList.java

```diff
@@ -273,4 +273,11 @@ public int size() {
 		}
 		return parameters.size();
 	}
+
+	public List<String> getParameterNames() {
```

Comment:
Namen hoeven niet uniek te zijn, misschien handig dat hier nog even bij te zetten? Of komen ze dan tweemaal in de lijst (ook dan handig te noteren)?